src/libraries/System.Speech/pkg/System.Speech.pkgproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Speech.csproj">
+      <SupportedFramework>netcoreapp2.1;net45;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Speech.csproj" />
+    <InboxOnTargetFramework Include="net45">
+      <AsFrameworkReference>true</AsFrameworkReference>
+    </InboxOnTargetFramework>
+    <File Include="$(PlaceHolderFile)">
+      <TargetPath>runtimes/win/lib/net45</TargetPath>
```

Comment:
@ericstj this would only be required if we had a netstandard2.0-windows configuration which we don't have here, so presumably we can just remove this placeholder right?

Reply:
Correct, this placeholder is unnecessary.  To elaborate: we need placeholders to accommodate the case where someone specifies a RID and a different TFM asset might apply.

Since the src project here targets `<TargetFrameworks>$(NetCoreAppCurrent)-windows;netcoreapp2.1-windows;netstandard2.0</TargetFrameworks>` even when someone targets `net48` and specifies a RID of `win`, no RID-specific assets will apply, only the `lib/net45/_._`

Reply:
Oh, missed this. Will fix.